.github/ISSUE_TEMPLATE/spec.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Spec
+about: Highlight a problem with a published or adopted spec
+title: ''
+labels: 'spec'
+assignees: ''
+
+---
+
+**Which spec or proposal does this relate to?**
```

Comment:
Under what circumstances would this be used for a proposal as opposed to using proposal.md?

Reply:
Given this, could the frontmatter line above be updated to clarify this?
```
about: Highlight a problem with a published or adopted spec
```

Maybe it could read something more like this?  What's the difference between a published and an adopted spec?
```
about: Highlight a problem with a published spec or accepted proposal
```

Or does this just complicate things?  Perhaps if proposal and spec documents are separate, then the first template would be for any proposal document no matter what the state, and this template would only be for spec documents.

Also, I see elsewhere in the Lifecycle->Accepted section of `Process.md`, it says: `Once a feature is accepted it becomes a planned feature for the release.`  Could `for the release` instead read `for a future release`?  That should eliminate the need to walk back from this, explaining that it may not "ship in the planned release" etc...  I'm not quite sure what "the release" refers to in the first place, unless each feature proposal will get a tagged for a specific release before being "accepted".

Reply:
I intended `for the release` to refer to the `Planned Version` in the proposal doc. We can always defer a proposal either by putting it into the **deferred** state or by updating it to a different `Planned Version`.